packages/quantic/force-app/main/default/lwc/quanticRecommendationList/quanticRecommendationList.js

```diff
@@ -77,6 +85,12 @@ export default class QuanticRecommendationList extends LightningElement {
    * @default {3}
    */
   @api headingLevel = 3;
+  /**
+   * The variant of the component. Accepted variants are ‘grid’ and carousel.
```

Comment:
Same comment as above about using back-ticks 🙂 
```suggestion
   * The variant of the component. Accepted variants are `grid` and `carousel`.
```